### PR TITLE
use rocksdb block cache speedup read

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by ngaut** [comment]
PTAL @siddontang @BusyJay

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -172,6 +172,13 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
                                        Some(64 * 1024),
                                        |v| v.as_integer());
     block_base_opts.set_block_size(block_size as u64);
+    let block_cache_size = get_integer_value("",
+                                             "rocksdb.block-based-table.block-cache-size",
+                                             matches,
+                                             config,
+                                             Some(1024 * 1024 * 1024),
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: 1G seems too big.

**Comment by siddontang** [comment]
@zhangjinpeng1987 tells me that this is for max cache size, so I think ok.

LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**